composer.json

```diff
@@ -27,7 +27,8 @@
     "sokil/php-isocodes": "^3.0",
     "myclabs/php-enum": "^1.7",
     "softonic/graphql-client": "^1.2",
-    "hashids/hashids": "^4.0"
+    "hashids/hashids": "^4.0",
+    "laravel/scout": "^7.2.1"
```

Comment:
This was the latest version that works with the current version of Laravel (v5.5) used in Rogue.